v2v/tests/src/v2v_options.py

```
@@ -786,7 +793,15 @@ def check_alloc():
             if not utils_v2v.v2v_supported_option("-ip <filename>"):
                 output_option = output_option.replace(
                     '-ip', '--password-file', 1)
-
+            # For VDDK
+            if input_transport == 'vddk':
+                vddk_thumbprint = utils_v2v.get_vddk_thumbprint(remote_host, source_pwd, 'vpx')
+                with tempfile.TemporaryDirectory(prefix='vddklib_') as vddk_libdir:
+                    utils_misc.mount(vddk_libdir_src, vddk_libdir, 'nfs')
+                    process.run('mkdir /home/vddk_libdir;cp -R %s/* %s' % (vddk_libdir, '/home/vddk_libdir'),
```

Ruff (0.14.2)

Function call with `shell=True` parameter identified, security issue

(S604)